src/components/RandomProductsBanner/RandomProductsBanner.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+$product-item-default-price-margin: 5px 0;
```

Comment:
I think you will want to have a default margin in common styles to be reused here and in all the app instead of 5px here.